SAKPaaS/src/app/components/sidenav/sidenav.component.html

```diff
@@ -0,0 +1,87 @@
+<div id="burger-menu" (click)="open()">
+  <mat-icon>menu</mat-icon>
+</div>
+
+<nav [class.open]="open$ | async" (swipeleft)="close()">
+  <div class="top">
+    <div class="user-header" *ngIf="authService.getUser() | async as user; else anonymous">
+      <p class="greeting">{{'user-card.greeting' | translate}} {{user.firstName}}!
+        <button mat-icon-button (click)="authService.logout()">
+          <mat-icon>highlight_off</mat-icon>
```

Comment:
weiß nicht, ich finde das Icon irgendwie irreführend, das sieht für mich aus, alsob ich damit die sidenav schließen würde.

Material Icons haben aber auch kein logout icon. Vielleicht kann man ja ein anderes einbinden (zB. https://www.flaticon.com/free-icon/logout_992680?term=logout&page=1&position=1)?

Reply:
Joa da gebe ich dir recht, nur wollte ich nicht den Struggle in Angriff nehmen, irgendeine externe SVG zusätlich einzubinden. Und selbst das Logout-Icon könnte ja "gehe raus aus diesem Menü" bedeuten, oder? Alternativ halt der Text "Abmelden"

Reply:
also ich kenne das mit der Tür und dem Pfeil nur als "Logout". In Figma is glaube auch nur n Text vorgesehen.

Aber soll mir jetzt erstmal wumpe sein, der Designer hat es ja so abgenommen :D